plugins/slack_app/config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Samson::Application .routes.draw do
+  get '/slack/oauth', to: 'slack_app#oauth'
```

Comment:
I checked, it's using [`/slack_webhooks`](https://github.com/zendesk/samson/blob/ben/slash-command/plugins/slack_webhooks/config/routes.rb#L2). But matching the namespace to the plugin name is probably better.